.gitignore

```diff
@@ -40,6 +40,7 @@
 ################################################
 
 config/local.js
+.env.local
```

Comment:
This lets you write your own .env.local file in the root directory in order to for example define your local ip address as the `VITE_API_URL` so you can view the app on other devices on your LAN without building the frontend